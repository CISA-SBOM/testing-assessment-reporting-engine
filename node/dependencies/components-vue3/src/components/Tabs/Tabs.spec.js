import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Tabs.vue'

describe('Tabs', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {
      directives: {
        'uid': {
          created(el) {
            el.setAttribute('id', 'unique-id')
          }
        }
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
