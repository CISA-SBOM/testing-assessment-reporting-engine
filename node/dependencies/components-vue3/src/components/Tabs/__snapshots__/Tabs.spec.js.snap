// Vitest Snapshot v1

exports[`Tabs > should match its default snapshot 1`] = `
<div
  data-id="sds-tabs"
  id="unique-id"
>
  <div
    class="overflow-x-auto bg-gray-100 dark:bg-gray-700 rounded-t"
  >
    <ul
      class="flex whitespace-nowrap z-10"
      role="tablist"
    >
      
      
    </ul>
  </div>
  
  
</div>
`;
